release/mixer.sh

```diff
@@ -78,20 +78,26 @@ build_update() {
 }
 
 build_deltas() {
+    local mix_format="${1}"
+
     section "Deltas"
 
-    log "Building delta packs..." 1
-    if (( NUM_DELTA_BUILDS > 0 )); then
-        sudo_mixer_cmd build delta-packs --previous-versions "${NUM_DELTA_BUILDS}"
+    local first_file="update/www/version/format${mix_format}/first"
```

Comment:
This should work even if the file does not exist, but needs to be cat instead of <
```bash
local first_version=$(cat "update/www/version/format${mix_format}/first" 2>/dev/null) || true
if [[ -n  "${first_version}" ]]; then
   # do delta
else
   # skip msg
fi
```

Reply:
Hmm, I'm a little confused what we are looking to do with the cat instead of <?